WcaOnRails/config/locales/cs.yml

```diff
@@ -2284,7 +2284,7 @@ cs:
         #original_hash: abd2828
         name: WCA Board of Directors
         #original_hash: d855b50
-        info_html: >-
+        description: >-
```

Comment:
This keeps the pattern of `name` and `description` that every other team has. I think it's ok not to have this *strong* since none of the other descriptions are. What do you think?

Reply:
Sounds good to me, so there are two ways you can go about this:
1. Update only the `en.yml` changing the key name and removing the `strong` tag. This will require other translators to translate the description again (without the `strong` tag).
2. Update all files changing the key name and removing the `strong` tag. This saves our translators the work, but requires more work from you.
Whichever you prefer =)

Reply:
Ok, I'm happy to do a little more work =). It's my understanding that translation changes are automatically detected using the previous hash- do I need to update the hash somehow?

Reply:
If you remove the strong tags from `en.yml`, so that the original text changes, then indeed it will be detected as outdated. But if you also remove `#original_hash` from the translation files, then it will fit the scenario "a new key as well as its translation has been added", thus will work as we want.
tl;dr; remove `#original_hash`es when going for `2.` =)